lib/skunk/cli/options/argv.rb

```diff
@@ -9,6 +9,12 @@ class Options
       # Extends RubyCritic::Cli::Options::Argv to parse a subset of the
       # parameters accepted by RubyCritic
       class Argv < RubyCritic::Cli::Options::Argv
+        attr_reader :output_filename
+
+        def initialize(argv)
+          super
+        end
```

Comment:
@manuca Is this necessary? If so, why?

Reply:
No, completely redundant. Seemed to be a remnant of something I tested at one point.